SimplaeWAWS.Models/ApiApp.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Newtonsoft.Json;
```

Comment:
When you moved these files here, were there any changes made to them? I can't tell since they all appear as new in the PR